doc/workflows_and_tools/tools/index.rst

```diff
@@ -31,6 +31,281 @@ a glimpse in the connection to your user store, you could as well in a first
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key tokenkind --tokeninfo-value software
+
+Search for all tokens with the tokeninfo-key ``tokenkind`` and the associated tokeninfo-value ``software``.
+
+tokeninfo-value-less-than INTEGER
+.................................
+
+Interpret tokeninfo-values as integers and match only if they are smaller than the given integer.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key timeWindow --tokeninfo-value-less-than 200
+
+Search for all tokens with the tokeninfo-key ``timeWindow`` and the associated tokeninfo-value below ``200``.
+
+tokeninfo-value-greater-than INTEGER
+....................................
+
+Interpret tokeninfo-values as integers and match only if they are greater than the given integer.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key timeWindow --tokeninfo-value-greater-than 100
+
+Search for all tokens with the tokeninfo-key ``timeWindow`` and the associated tokeninfo-value greater than ``100``.
+
+Actions
+*******
+
+Actions that can be performed on the found tokens.
+
+mark - disable - delete - unassign - export - listuser - tokenrealms
+
+mark
+....
+
+**Mark** makes it possible to mark single or multiple tokens in order to carry out further actions with them later.
+
+Please note that without a previous selection of a certain token, all found tokens are automatically marked.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action mark
+
+The token with the serial ``OAUTH0004C934`` will be marked.
+
+
+disable
+.......
+
+With **disable** single or multiple tokens can be disabled.
+
+Please note that without a previous selection of a certain token, all found tokens will be disabled.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action disable
+
+The token with the serial ``OAUTH0004C934`` will be disabled.
+
+delete
+......
+
+With **delete** single or multiple tokens can be deleted.
+
+Please note that without a previous selection of a certain token, all found tokens will be deleted.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action delete
+
+The token with the serial ``OAUTH0004C934`` will be deleted.
+
+export
+......
+
+With **export** single or multiple tokens can be exported as csv or pskc.
+
+Please note that without a previous selection of a certain token, all found tokens will be exported.
+
+Export only possible with HOTP and TOTP token.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action export > OAUTH0004C934.xml
+
+The token with the serial ``OAUTH0004C934`` will be exported and saved in an xml file.
+
+Note that you need your encryption key for re-import.
+
+listuser
+........
+
+With **listuser** the various users and their tokens are listed in a summarized view.
+
+Example::
+
+    privacyidea-token-janitor find --action listuser
+
+lists all users and their tokens together.
```

Comment:
You can not list all users! (see above)

You can list all tokens. This is a difference. The key is the token not the user.